src/audio/fft/twiddle.h

```diff
@@ -0,0 +1,2073 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2017 Intel Corporation. All rights reserved.
+ *
+ * Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+ *	   Amery Song <chao.song@intel.com>
+ *	   Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+/* Twiddle factors in Q1.31 format */
+
+#ifndef __INCLUDE_TWIDDLE_H__
```

Comment:
why is the coeff data in a header ? @singalsu are we doing this for SRC etc ?
Btw header include should match the path name.

Reply:
The SRC coefficients are in include/sof/audio/coefficients , maybe these could go there. If the computation is simple and there would be several sets they could be computed in some fft_init() function. The SRC coefficients are take long to compute and require hard to implement functions so they are pre-computed.

Reply:
ack, they should all go in one place.